core/templates/pages/exploration-editor-page/param-changes-editor/param-changes-editor.component.spec.ts

```diff
@@ -45,6 +45,10 @@ import { ParamSpecsObjectFactory } from
 import { StateCustomizationArgsService } from
   // eslint-disable-next-line max-len
   'components/state-editor/state-editor-properties-services/state-customization-args.service';
+import { destroyPlatform } from '@angular/core';
+import { setupAndGetUpgradedComponent } from 'tests/unit-test-utils';
+import { async } from '@angular/core/testing';
+import { ParamChangesEditorDirective } from './param-changes-editor.component';
```

Comment:
The import order we follow is this:

1. Angular imports (imports from @angular/*)
2. Third-party imports (imports from rxjs, ng bootstrap, etc)
3. Application imports. (imports from our application)

Within these sub-categories: sort lexicographically based on module (i.e. where it is imported from, not what is being imported).
For example:
#### Right way (because 'B' < 'D')
```
import { C } from 'B';
import { A } from 'D'; 
```

#### Wrong way
```
import { A } from 'D';
import { C } from 'B'; 
```

Can you check for that here and every other "migrated" file (not wrapped).

Reply:
Got you!

Reply:
Done!